src/include/parser/explain_statement.h

```diff
@@ -0,0 +1,40 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// explain_statement.h
+//
+// Identification: src/include/parser/explain_statement.h
+//
+// Copyright (c) 2018, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "common/logger.h"
+#include "common/sql_node_visitor.h"
+#include "expression/abstract_expression.h"
+#include "parser/sql_statement.h"
+#include "parser/table_ref.h"
+
+namespace peloton {
+namespace parser {
+
+/**
+ * @class ExplainStatement
+ * @brief Represents "EXPLAIN <query>"
+ */
+class ExplainStatement : public SQLStatement {
+ public:
+  ExplainStatement()
+      : SQLStatement(StatementType::EXPLAIN) {}
+  virtual ~ExplainStatement() {}
+
+  void Accept(SqlNodeVisitor* /*v*/) override {}
+
+  std::unique_ptr<parser::SQLStatement> real_sql_stmt;
+};
```

Comment:
1. Has this file been formatter using the `formatter.py` script?
2. Either add in the variable name in `Accept()` with an `UNUSED_ATTRIBUTE` or remove it altogether.